template/koa-rest-api/src/framework/validation.ts

```diff
@@ -51,11 +70,13 @@ export const validate = <
 }): Output => {
   const parseResult = schema.safeParse(input);
   if (parseResult.success === false) {
+    const invalidFields: InvalidFields = {};
+    parseInvalidFieldsFromError(invalidFields, parseResult.error);
```

Comment:
Nitpick: can we keep `parseInvalidFieldsFromError` a pure function that constructs its own response rather than it mutating a parameter?